ee/tabby-webserver/src/hub/api.rs

```diff
@@ -104,10 +107,21 @@ impl CodeSearch for HubClient {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct ClientRequest {
+    pub port: u16,
```

Comment:
port should be only relevant to RegisterWorkerRequest. For job, it doesn't requires a local server.